cuenca/http/client.py

```diff
@@ -106,17 +106,20 @@ def get(
         self,
         endpoint: str,
         params: ClientRequestParams = None,
+        **kwargs,
     ) -> DictStrAny:
-        return self.request('get', endpoint, params=params)
+        return self.request('get', endpoint, params=params, **kwargs)
 
-    def post(self, endpoint: str, data: DictStrAny) -> DictStrAny:
-        return self.request('post', endpoint, data=data)
+    def post(self, endpoint: str, data: DictStrAny, **kwargs) -> DictStrAny:
```

Comment:
Why?

Reply:
The request method already receives a kwargs, I just send it to them.

To download the pdf we need to send the header with Accept and the stream field that indicates when you will get the data from the file.

<img width="547" alt="Screen Shot 2020-12-21 at 14 38 05" src="https://user-images.githubusercontent.com/25915735/102820124-5b41de00-439a-11eb-81ad-07c0740c3119.png">

Reply:
At some point a post may need headers or some other field.